### PR TITLE
Remove rsync from build step

### PR DESCRIPTION
**Pull request by kandl**
- replace with cp and rm commands

JIRA: RAIL-4047

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-create-gooddata-react-app-unit-tests-zuul-docker-el7/374/ : SUCCESS in 1m 06s
- https://checklist.intgdc.com/job/compliance/job/gooddata-create-gooddata-react-app-oss-check/294/ : SUCCESS in 14s

File: scripts/build.sh [hunk 1]
```diff
@@ -37,14 +38,10 @@ npm-do prettier --write "build/tar-source/**/*" \
...
-rsync -rv \
-  --exclude='node_modules' \
-  --exclude='build' \
-  --exclude='tsconfig.json' \
-  --exclude='*.ts' \
-  --exclude='*.tsx' \
-  bootstrap/ \
-  build/tar-source
+cp -rv bootstrap/ build/tar-source
+rm -rf build/tar-source/**/**.ts
```
Comment (no23reason) [inline scripts/build.sh:42]: Should end with just *.ts, not **.ts

**Review by no23reason — APPROVED** [review]
LGTM

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-create-gooddata-react-app-merge-compliance-check/154/ : SUCCESS in 19s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-create-gooddata-react-app-unit-tests-zuul-docker-el7/376/ : SUCCESS in 56s
- https://checklist.intgdc.com/job/compliance/job/gooddata-create-gooddata-react-app-fossa-scanning-zuul-el7/119/ : SUCCESS in 43s (non-voting)

**Pull request merged**